extensions/example-extension/renderer.tsx

```diff
@@ -30,13 +55,48 @@ export default class ExampleExtension extends LensRendererExtension {
     }
   ];
 
-  clusterPageMenus: Interface.PageMenuRegistration[] = [
+  clusterPageMenus: Interface.ClusterPageMenuRegistration[] = [
     {
+      id: "top-example-menu",
       title: "Example extension",
       components: {
         Icon: ExampleIcon,
       },
     },
+    {
+      parentId: "top-example-menu",
+      title: "Simple Page",
+      target: {
+        pageId: "simple-page"
+      },
+      components: {
+        Icon: ExampleIcon,
+      },
+    },
+    {
+      parentId: "top-example-menu",
+      title: "Simple Params",
+      target: {
+        pageId: "page-with-simple-parms",
+        params: {
+          exampleId: "no-secret",
+          namespace: "test"
+        }
+      },
+      components: {
+        Icon: ExampleIcon,
+      },
+    },
+    {
+      parentId: "top-example-menu",
+      title: "Complicated Params",
+      target: {
+        pageId: "page-with-complicated-params"
+      },
+      components: {
+        Icon: ExampleIcon,
+      },
+    },
```

Comment:
this is your original page, it works if you comment out the `parentId`.

I did notice an issue (before this PR too). the first time the page is rendered the location url is missing the `selectedNamespaces` param:
![image](https://user-images.githubusercontent.com/40840436/102418168-32e96680-3fcb-11eb-86a3-c1cd1bd3429e.png)

Reply:
It's not missing, used `location.href` non observable and `selectedNamespaces` therefore not loaded yet when you don't see it.

Reply:
What can be displayed here that is observable? I'm not sure it's meaningful to show this if it doesn't represent the params in the example

Reply:
Yes, could be hidden. Anyway don't put too much attention to example itself since it's not used anywhere. 
Just  for playing around / demo purposes.